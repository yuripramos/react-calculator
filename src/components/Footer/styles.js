import styled, { css } from "styled-components";
import { rem, media } from "../../styles/tools";
import { grey10, black } from "../../styles/settings";

export const ContentWrapper = styled.div`
  padding: ${rem(15)} 0;
  background: ${black}
`;

export const Signature = styled.div`
  color: ${grey10};
  font-family: "OpenSans Thin";
  font-size: ${rem(10)};
  display: inline-flex;
  align-items: center;
  justify-content: center;
  margin-top: ${rem(5)};
  width: 100%;
  ${media.sm(css`
    font-size: ${rem(12)};
  `)};
  ${media.md(css`
    font-size: ${rem(14)};
  `)};
  ${media.lg(css`
    font-size: ${rem(16)};
  `)};
`;

