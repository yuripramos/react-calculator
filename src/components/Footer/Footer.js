import React from "react";
import { Signature, ContentWrapper } from "./styles";
import { Container, Row, Column } from "../../styles/grid";

function Footer() {
  return (
    <ContentWrapper>
      <Container>
        <Row />
        <Row>
          <Column>
            <Signature>made with ðŸ’› by Yuri Ramos</Signature>
          </Column>
        </Row>
      </Container>
    </ContentWrapper>
  );
}

export default Footer;
