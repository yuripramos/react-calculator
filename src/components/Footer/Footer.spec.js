import React from "react";
import { shallow } from "enzyme";
import Footer from "./Footer";

describe("Footer component", () => {
  it("should match snapshot", () => {
    expect(shallow(<Footer />)).toMatchSnapshot();
  });
});
