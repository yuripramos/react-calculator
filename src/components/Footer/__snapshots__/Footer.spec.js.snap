// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should match snapshot 1`] = `
<styled.div>
  <Container>
    <Row />
    <Row>
      <Column>
        <styled.div>
          made with ðŸ’› by Yuri Ramos
        </styled.div>
      </Column>
    </Row>
  </Container>
</styled.div>
`;
